bullet_train-outgoing_webhooks/test/integration/api/a1/webhooks/outgoing/endpoints_controller_test.rb

```diff
@@ -3,6 +3,17 @@
 class Api::V1::Webhooks::Outgoing::EndpointsControllerTest < ActionDispatch::IntegrationTest
   setup do
     @controller = Api::V1::Webhooks::Outgoing::EndpointsController
+
+    5.times do |n|
+      Team.create! name: "Generic name #{n}"
+    end
+
+    5.times do |n|
+      Webhooks::Outgoing::Endpoint.create! team_id: Team.first.id, name: "Generic name #{n}", url: "http://example.com/webhook-#{n}"
+    end
```

Comment:
Moved these into the `setup` method so that they're created within the transaction for running tests. When they were outside of the `setup` method in the `setup/active_record.rb` file the records created would stay in the DB between test runs.